code/modules/reagents/reagent_containers/cups/organ_jar.dm

```diff
@@ -1,37 +1,34 @@
-// The organ jar - a 150u bottle that can hold a single organ
-/obj/item/reagent_containers/cup/organ_jar
+// The organ jar - a 120u beaker that can hold a single organ
+/obj/item/reagent_containers/cup/beaker/organ_jar
```

Comment:
This needs an update typepath script

Reply:
Ostensibly it doesn't need a script if it's not mapped in anywhere